ui/css/treeherder.css

```diff
@@ -88,8 +88,8 @@ input:focus::-moz-placeholder {
 
 .th-top-nav-options-panel {
     background-color: white;
-    padding: 10px;
-    border: 1px solid black;
+    padding: 10px 10px 0;
```

Comment:
So basically here we are removing the bottom padding on the parent panel, and adding an equivalent `exclusion-container` class of the same size (see L161 below) for child elements.